### PR TITLE
[PasswordHasher] Mention standalone use of `PasswordHasherFactory`

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fixes https://github.com/symfony/symfony-docs/issues/18020

### CODE REVIEW

File: security/passwords.rst [hunk 1]
```diff
@@ -697,6 +697,32 @@ you must register a service for it in order to use it as a named hasher:
 This creates a hasher named ``app_hasher`` from a service with the ID
 ``App\Security\Hasher\MyCustomPasswordHasher``.
 
+Hashing a Stand-Alone String
+----------------------------
+
+The password hasher can be used to hash strings independently
+of users. By using the
+:class:`Symfony\\Component\\PasswordHasher\\Hasher\\PasswordHasherFactory`,
+you can retrieve declare multiple hashers, retrieve any of them with
```
Comment (94noni) [inline security/passwords.rst]: `retrieve declare` next to each other seems weird when reading, 2 verbs 
is there a word missing in between ?

File: security/passwords.rst [hunk 2]
```diff
@@ -697,6 +697,32 @@ you must register a service for it in order to use it as a named hasher:
...
+    $factory = new PasswordHasherFactory([
+        'common' => ['algorithm' => 'bcrypt'],
+        'sodium' => ['algorithm' => 'sodium'],
+    ]);
+
+    // retrieve the hasher using bcrypt
+    $passwordHasher = $factory->getPasswordHasher('common');
+    $hash = $passwordHasher->hash('plain');
+
+    // verify that a given plain password matches the hash calculated above
```
Comment (94noni) [inline security/passwords.rst]:
```suggestion
    // verify that a given plain string matches the hash calculated above
```

as it seems you want to document the hashing process of a bare string, not password string

File: security/passwords.rst (hunk 1 above)
Comment (alexandre-daubois) [reply security/passwords.rst]: Fixed!

File: security/passwords.rst (hunk 2 above)
Comment (alexandre-daubois) [reply security/passwords.rst]: Fixed as well, thanks!

**Review by 94noni — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @alexandre-daubois.

**Pull request merged**